assets/dev/scss/frontend/_div-block.scss

```diff
@@ -0,0 +1,6 @@
+// TODO remove in feature [ED-16336]
+.div-block {
+	display: block;
+	padding-block-start: var(--padding-block-start);
+	padding-block-end: var(--padding-block-end);
```

Comment:
Where do these variables come from?

Reply:
assets/dev/scss/frontend/_container.scss

Reply:
Shouldn't depend on the V1 cntainer